core/tests/test_utils_test.py

```diff
@@ -428,13 +428,6 @@ def mock_getcwd():
 class EmailMockTests(test_utils.EmailTestBase):
     """Class for testing EmailTestBase."""
 
-    def test_override_run_swaps_contexts(self):
```

Comment:
Any reason for dropping this?

Reply:
Couldn't really figure out a clean way of checking for function swap change so I thought it would suffice to use the second function(it wouldn't work if the function swap didn't occur). 

However, with further thought, I took some inspiration from the way self.swap was implemented and made a check for the function swap.